Networking/Networking/Remote/CouponsRemote.swift

```diff
@@ -87,7 +88,8 @@ public final class CouponsRemote: Remote, CouponsRemoteProtocol {
                                      method: .get,
                                      siteID: siteID,
                                      path: Path.coupons,
-                                     parameters: parameters)
+                                     parameters: parameters,
+                                     availableAsRESTRequest: true)
```

Comment:
I was wondering why `availableAsRESTRequest: true` isn't applied to two requests, `retrieveCoupon` and `deleteCoupon`?

Reply:
Oh, that was a mistake. 🤦 

Fixed it in b1884f0746823610d3a8a32b19685b64b7fea9c0